fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -154,21 +155,40 @@ private String findNearestParent(final String childPath) {
     private boolean isAuthorized(final Subject currentUser, final HttpServletRequest httpRequest) throws IOException {
         final String requestURL = httpRequest.getRequestURL().toString();
         final String repoPath = httpRequest.getPathInfo();
-
+        final boolean isAcl = requestURL.endsWith(FCR_ACL);
         final URI requestURI = URI.create(requestURL);
         log.debug("Request URI is {}", requestURI);
 
         // WebAC permissions
         final WebACPermission toRead = new WebACPermission(WEBAC_MODE_READ, requestURI);
         final WebACPermission toWrite = new WebACPermission(WEBAC_MODE_WRITE, requestURI);
         final WebACPermission toAppend = new WebACPermission(WEBAC_MODE_APPEND, requestURI);
+        final WebACPermission toControl = new WebACPermission(WEBAC_MODE_CONTROL, requestURI);
```

Comment:
I think the requestURI for the `toControl` permission needs to be the resource, *not* the ACL. I think that is what is causing the test in step 8 of your verification steps to fail.

Reply:
@peichman-umd :  I ran through the tests again.  It looks like everything is working.  I have pushed up those changes as well as a tangentially related issue with Content-Types (that I ran into while writing CTS tests for WebAC).

Reply:
@dbernstein I am getting 403 responses for everything from step 8 onward.

Reply:
@peichman-umd :  I just ran through it again (third time) and everything worked correctly.  Perhaps you are not running off of a clean repo?  Anyway,  given that @awoods was able to run the tests, what do you say you just merge it.  Between the integration tests and the CTS tests any remaining bugs should become apparent.